dotcom-rendering/makefile

```diff
@@ -95,6 +95,11 @@ dev-legacy: clear clean-dist install gen-dotenv
 
 cypress: export NODE_ENV=production
 cypress: clear clean-dist install build
+	$(call log, "starting frontend PROD server for Cypress")
+	@DISABLE_LOGGING_AND_METRICS=true start-server-and-test 'node dist/frontend.server.js' 9000 'cypress run --browser chrome --spec "cypress/e2e/**/*"'
+
+cypress-arm64: export NODE_ENV=production
+cypress-arm64: clear clean-dist install build
```

Comment:
New makefile Cypress task specifically for TeamCity / ARM64 that runs with the default Electron browser

Reply:
Is there no way around that? The number of environments we have seems to keep growing all the time…

Reply:
Could be worth noting in the file - maybe noting that we should deprecate this. I've added it to this issue: